pkgs/by-name/vu/vulkan-hdr-layer-kwin6/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  fetchgit,
+  libX11,
+  meson,
+  ninja,
+  pkg-config,
+  vulkan-headers,
+  vulkan-loader,
+  wayland-scanner,
+  wayland,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vulkan-hdr-layer-kwin6";
+  version = "0-unstable-2024-10-19";
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    cmake
+  ];
+
+  buildInputs = [
+    vulkan-headers
+    vulkan-loader
+    libX11
+    wayland-scanner
+    wayland
+  ];
+
+  src = fetchgit {
```

Comment:
You could also use fetchFromGitHub, but it doesn't matter much.

Reply:
Thanks, updated.